test/e2e/cypress/integration/hosts_overview.js

```diff
@@ -3,6 +3,9 @@ import { allHostNames, agents } from '../fixtures/hosts-overview/available_hosts
 context('Hosts Overview', () => {
     const availableHosts = allHostNames()
     before(() => {
+        cy.exec(`${Cypress.env('trento_binary')} ctl db-reset --db-host=${Cypress.env('db_host')} --db-port=${Cypress.env('db_port')}`)
+        cy.exec(`cd ${Cypress.env('fixtures_path')} && ${Cypress.env('photofinish_binary')} run --url "http://${Cypress.env('collector_host')}:${Cypress.env('collector_port')}/api/collect" healthy-SAP-cluster-27-nodes`)
+        
```

Comment:
We could refactor this in a utility function to be shared between the suites, the function could take the scenario name as a parameter (e.g. "healthy-SAP-cluster-27-nodes").
This would come handy when adding the new tests.

Reply:
Yes, makes sense :+1:

Reply:
Made a couple of Cypress commands:
- `resetDatabase`
- `loadScenario`

That can be used like this
```
cy.resetDatabase()
cy.loadScenario('healthy-27-node-SAP-cluster')
```

Sounds good?

Reply:
I love it :heart:

Reply:
Great!